docs/markdown/Python-module.md

```diff
@@ -37,7 +37,10 @@ If provided, it can be:
 - One of `python2` or `python3`: in either case, the module will try
   some alternative names: `py -2` or `py -3` on Windows, and `python`
   everywhere. In the latter case, it will check whether the version
-  provided by the sysconfig module matches the required major version
+  provided by the sysconfig module matches the required major version.
+
+  *Since 1.2.0*, you can also search for more specific python version
+  on Windows, using `python3.11`. 
```

Comment:
I see the `version` keyword as something different. On my (Windows) system, I have many different python versions installed (3.9, 3.10, 3.11). Having a `version` keyword would allow to ensure the found version matches a specification, but would not allow my to specify the exact version I need.

On my Linux systems, there are symlinks like python3.11. Therefore, I thought it was standard.

Reply:
How do non-Windows systems manage the parallel installation of many python versions if there is no python3.xx symlinks?

Reply:
> How do non-Windows systems manage the parallel installation of many python versions if there is no python3.xx symlinks?

You can have versioned symbolic links, if you have multiple interpreters installed, but they are not required to exist.

Reply:
> * The Python build system for Unix always installs the versioned program, and optionally creates symlinks called "python3" and "python". The canonical executable name for the exact version of python after symlink resolution is _always_ pythonX.YY, even on Unix. No matter how many versions you have installed.

Right. Sorry. I don't know why but I was convinced that Debian does not do this by default, but it does.

>     * The `version:` kwarg for meson restricts the permitted versions as a comparison specifier, compatible with `string.version_compare()`. Exact specifiers such as `version: '3.10'` are permitted; more common is to use >= or other range operators. Version specifiers accept arrays; all comparison specifiers must succeed. While it is possible to specify an exact version, this is unlikely to be what people want in most cases.

Indeed. That's why I think having a `version` keyword argument is more flexible than getting the version from the executable name.

Reply:
I still think it does the right thing. I see the `version` keyword as a complementary feature. I will probably try to implement it too, but I think this is a more complex task to do.

My use case is that I want reproductible builds. Therefore, I need to fix the Python version I link to (even the patch number must be fixed), to prevent a Python upgrade to break my build. My project is a monorepo, where there can be different applications working on different Python versions. Therefore, I need to be able to link with different specific python versions (e.g. python3.10, python3.11, python2.7...) for different python modules.

Reply:
If it works correctly on all platforms, the addition to the documentation seems at least misleading to me. I would interpret it as meaning that specifying the version in the executable name works *only* on Windows.

Reply:
I updated the documentation.